### ISSUE TITLE
Leaf-kit 1.0.0-rc.1.17 no longer compiling on raspberry pi

### ISSUE THREAD

**Issue by bottlehall** [vapor/leaf-kit#69]
Compile generates an error due to line 45 of file LeafKit/LeafData/LeafDataRepresentable.swift. The offending line is:
```swift
extension Float80: LeafDataRepresentable {}
```

Enclosing it in compiler directives fixes the problem:
```swift
#if arch(i386) || arch(x86_64)
extension Float80: LeafDataRepresentable {}
#endif
```

**Comment by tdotclare** [comment]
Will patch in next release.

**Comment by 0xTim** [comment]
This also occurs on Apple Silicon

**Comment by tdotclare** [comment]
Closed in LeafKit-1.0.0tau
